public/app/core/components/AppChrome/DockedMegaMenu/MegaMenu.tsx

```diff
@@ -60,7 +60,7 @@ export const MegaMenu = React.memo(
         </div>
         <nav className={styles.content}>
           <CustomScrollbar showScrollIndicators hideHorizontalTrack>
-            <ul className={styles.itemList}>
+            <ul className={styles.itemList} aria-label={t('navigation.megamenu.listLevel1', 'navigation')}>
```

Comment:
maybe `list-label` instead of `listLevel1`? also think `Navigation` should be capitalised